cmd/tools/zapi/zapi.go

```diff
@@ -251,11 +246,13 @@ func getCounter(c *client.Client, args *Args) (*node.Node, error) {
 	}
 
 	for _, cnt = range counters.GetChildren() {
-		if cnt.GetChildContentS("name") == args.Counter {
-			return cnt, nil
+		for _, counter := range args.Counters {
+			if cnt.GetChildContentS("name") == counter {
+				return cnt, nil
```

Comment:
This would give counter info of only one value, not for all the provided in CLI.

Reply:
`./bin/zapi -p cluster-21 show data --object qtree --counter internal_ops,nfs_ops` is working fine but `./bin/zapi -p cluster-21 show counter --object qtree --counter instance_uuid,nfs_ops,internal_ops` is not.

Reply:
good catch